tests/mov.brain

```diff
@@ -0,0 +1 @@
++++++>++*&^#
```

Comment:
I may be blind or something, but tell me that there's 6 `+` instead of 5 @luizperes.

Reply:
Hey @rafaelcn, thanks for pointing that! Thank you so much, however, I checked and one of these pluses is from the github 
<img width="190" alt="screen shot 2018-01-31 at 11 59 43 am" src="https://user-images.githubusercontent.com/6173065/35644609-41b1ecac-067e-11e8-8a52-cc2e1681eb93.png">

It is in the same style and with no spacements, so we seems that it is part of de code xD

Reply:
Wow, that's awkward. Approved then, gonna merge it.